hs_core/page_processors.py

```diff
@@ -122,6 +122,7 @@ def get_page_context(page, user, resource_edit=False, extended_metadata_layout=N
     keywords = json.dumps([sub.value for sub in content_model.metadata.subjects.all()])
     topics = Topic.objects.all().values_list('name', flat=True).order_by('name')
     topics = list(topics)  # force QuerySet evaluation
+    content_model.update_relation_meta()
```

Comment:
Is there a better place to call update_relation_meta() than in get_page_context() which will be called frequently whenever a resource page is displayed?

Reply:
Since we need to display the up to date citation when displaying relation types on UI, I think it needs to be here.